Manufacturers.md

```
@@ -75,7 +80,8 @@ This is the official list of manufacturer ids (`manufacturer_id` in the target c
 |HNEC|Huizhou NIDICI Electronic Co., Ltd|https://shop.nidici.com/|
 |HOWI|Hobbywing Technology Co., Ltd|http://hobbywing.com/|
 |IFRC|iFlight Innovation Technology Ltd.|https://www.iflight-rc.com/|
-|JHEF|JHE\_FPV|https://www.jhemcu.com/|
+|JHEF|Chenzhen Jinhang Electronic Co., Ltd|https://www.jhemcu.com/|
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)